wificontrol/wpasupplicant.py

```diff
@@ -81,12 +84,45 @@ def started(self):
 
         return wpa_supplicant_started
 
+    def check_for_errors(self):
+        error_state = self.sysdmanager.is_failed("wpa_supplicant.service")
+        if error_state:
+            try:
+                return error_state["exit-code"]
+            except KeyError:
+                return error_state
```

Comment:
What can there be?

Reply:
There is method `get_error_code` now, which returns error code if service has failed state.